tsuru/client/services.go

```diff
@@ -615,26 +614,34 @@ func (c ServiceInfo) Run(ctx *cmd.Context, client *cmd.Client) error {
 }
 
 type ServiceInstanceRemove struct {
-	yes       bool
-	yesUnbind bool
-	fs        *gnuflag.FlagSet
+	cmd.ConfirmationCommand
+	fs    *gnuflag.FlagSet
+	force bool
 }
 
 func (c *ServiceInstanceRemove) Info() *cmd.Info {
 	return &cmd.Info{
 		Name:  "service-instance-remove",
-		Usage: "service-instance-remove <service-name> <service-instance-name> [--assume-yes] [--unbind]",
+		Usage: "service-instance-remove <service-name> <service-instance-name> [-f/--force] [-y/--assume-yes]",
 		Desc: `Destroys a service instance. It can't remove a service instance that is bound
 to an app, so before remove a service instance, make sure there is no apps
 bound to it (see [[tsuru service-instance-info]] command).`,
 		MinArgs: 2,
 	}
 }
 
-func removeServiceInstanceWithUnbind(ctx *cmd.Context, client *cmd.Client) error {
+func (c *ServiceInstanceRemove) Run(ctx *cmd.Context, client *cmd.Client) error {
 	serviceName := ctx.Args[0]
 	instanceName := ctx.Args[1]
-	url := fmt.Sprintf("/services/%s/instances/%s?unbindall=%s", serviceName, instanceName, "true")
+	msg := fmt.Sprintf("Are you sure you want to remove the instance %q", instanceName)
+	if c.force {
+		msg += " and remove all binds"
+	}
+	if !c.Confirm(ctx, msg+"?") {
+		return nil
+	}
+
+	url := fmt.Sprintf("/services/%s/instances/%s?unbindall=%s", serviceName, instanceName, strconv.FormatBool(c.force))
```

Comment:
Just a nitpick, you can use `unbindall=%v` instead of `%s` and then there's no need to using `strconv.FormatBool` you can use `c.force` directly.

Reply:
Great, thanks! :D